testsuite/replay_verify.py

```diff
@@ -16,52 +16,66 @@
 # This script runs the replay-verify from the root of aptos-core
 # It assumes the aptos-db-tool binary is already built with the release profile
 
+
+# The key is the runner's number and the value is the range of txns that the runner is responsible for
+# This mapping is generated in 3 steps:
+# (1) allocate the txns to runners based on how much time the runners in past takes to finish evenly distributed txns, example script https://gist.github.com/areshand/d00ce302d3bafe0f7b97c311113eba1b
+# (2) rerun the flow with new ranges and check the time each runner takes to finish again
+# (3) manual adjust the range based on times each takes to finish the work to make sure each runner takes similar time to finish
+
+#  Note: this range needs to be updated when the last range's time is over 2 hrs. (we will send a low pri alert once the time is over 2 hrs)
+#  Oncall should
+#  1. seal the last range with the latest txn version and start a new range with the [latest_txn_version + 1, sys.maxsize]
+#  2. meanwhile, the oncall should delete the old ranges that are beyond 300M window that we want to scan
+#
+
 testnet_runner_mapping = {
-    0:[250000000, 255584106],
-    1:[255584107, 271874718],
-    2:[271874719, 305009463],
-    3:[305009464, 324904819],
-    4:[324904820, 347234877],
-    5:[347234878, 366973577],
-    6:[366973578, 399489396],
-    7:[399489397, 430909965],
-    8:[430909966, 449999999],
-    9:[450000000, 462114510],
-    10:[462114511, 477825432],
-    11:[477825433, 485000000],
-    12:[485000001, 516281795],
-    13:[516281796, 551052675],
-    14:[551052676, 582481398],
-    15:[582481399, sys.maxsize]
+    0: [250000000, 255584106],
```

Comment:
make it
```
testnet_runner_mapping = [
    [255584107, 271874718],
    [271874719, 305009463],
    ...
```
?

Reply:
Ok, updated, I thought you wanted to remove strategy.matrix.number array

Reply:
both, I want ALL. 🚀